README.md

```diff
@@ -47,124 +59,269 @@
+</td>
+<td><h3>🐣</h3></td>
+
+<tr>
+<td>
+<h4><a href="https://github.com/jina-ai/examples/tree/master/multilingual-search">Multilingual semantic search with Laser</a></h4>
+Language-agnostic search on WMTData
+</td>
+<td><h3>🐣</h3></td>
+</tr>
 
 <tr>
 <td>
-<h4><a href="https://docs.jina.ai/chapters/flow/index.html">Use Flow API to Compose Your Search Workflow</a></h4>
-Learn how to orchestrate Pods to work together: sequentially and in parallel; locally and remotely
+<h4><a href="https://github.com/jina-ai/examples/tree/master/multires-lyrics-search">Search Lyrics with Transformers and PyTorch</a></h4>
+Get a better understanding of chunks by searching a lyrics database. Now with shiny front-end!
 </td>
 <td><h3>🐣</h3></td>
 </tr>
 
+</table>
+
+
+### Search Images
+
+<table width=100% style='width: 100%'>
+<tr>
+<th width="90%" style="width: 90%">Tutorials</th>
+<th width="10%" style="width: 10%">Level</th>
+</tr>
+<tr>
+<td>
+<h4><a href="https://github.com/jina-ai/examples/tree/master/flower-search">Search Flowers with TorchVision</a></h4>
+Build an image search system and define your own executors and run them in Docker
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+</td>
+<td><h3>🐣</h3></td>
+</tr>
+
+
+<tr>
+<td>
+<h4><a href="https://github.com/jina-ai/examples/tree/master/pokedex-with-bit">Google's Big Transfer Model in (Poké-)Production</a></h4>
+Use SOTA visual representation for searching Pokémon!
+</td>
+<td><h3>🚀</h3></td>
+</tr>
+
+<tr>
+<td>
+<h4><a href="https://github.com/jina-ai/examples/tree/master/mnist-with-mindspore">Search MNIST with Mindspore</a></h4>
+Search handwriting examples using Mindspore
+</td>
+<td><h3>🚀</h3></td>
+</tr>
```

Comment:
Let's move mindspore to `Extract Features and Vectors` and rename that section to `Misc`. I'm not sure about the name.